options/locale/locale_en-US.ini

```
@@ -1411,8 +1411,8 @@ issues.context.edit = Edit
 issues.context.delete = Delete
 issues.no_content = There is no content yet.
 issues.close = Close Issue
-issues.pull_merged_at = `merged commit <a class="ui sha" href="%[1]s"><code>%[2]s</code></a> into <b>%[3]s</b> %[4]s`
-issues.manually_pull_merged_at = `merged commit <a class="ui sha" href="%[1]s"><code>%[2]s</code></a> into <b>%[3]s</b> manually %[4]s`
+issues.comment_pull_merged_at = `merged commit %[1]s into %[2]s %[3]s`
+issues.comment_manually_pull_merged_at = `manually merged commit %[1]s into %[2]s %[3]s`
```

```suggestion
issues.comment_pull_merged_at = merged commit %[1]s into %[2]s %[3]s
issues.comment_manually_pull_merged_at = manually merged commit %[1]s into %[2]s %[3]s
```

Think we can omit these now.